backend/libraries/group_chat_core/src/members.rs

```diff
@@ -35,6 +35,7 @@ pub struct GroupMembers {
     moderators: BTreeSet<UserId>,
     bots: BTreeMap<UserId, UserType>,
     notifications_unmuted: BTreeSet<UserId>,
+    at_everyone_muted: BTreeSet<UserId>,
```

Comment:
`serde(default)`

Reply:
And let's get rid of the alias on line 30, we haven't needed that for ages.

Reply:
Oops